repo/blob/filesystem/filesystem_storage.go

```diff
@@ -70,6 +70,22 @@ func (fs *fsImpl) isRetriable(err error) bool {
 	return errors.Is(err, errRetriableInvalidLength)
 }
 
+func (fs *fsStorage) GetCapacity(ctx context.Context) (blob.Capacity, error) {
+	c, err := retry.WithExponentialBackoff(ctx, "GetCapacity", func() (interface{}, error) {
+		stat, err := fs.Impl.(*fsImpl).osi.Statfs("/")
```

Comment:
If I'm reading this right - this is checking statfs of the root filesystem - this may not be the filesystem of the actual repository backup right?

`/` might only have 1mb free, but `/data` may have 300gb?

Reply:
Yes, will fix.